dictionary-builder/src/main.rs

```diff
@@ -420,14 +323,9 @@ where
             vr2.to_string()
         };
 
-        let mut obs = obs.unwrap_or_else(String::new);
-        if obs != "" {
-            obs = format!(" // {}", obs.as_str());
-        }
-
         writeln!(
             f,
-            "    E {{ tag: {}, alias: \"{}\", vr: {}{} }},{}",
+            "    E {{ tag: {}, alias: \"{}\", vr: {}{} }}, // {}",
```

Comment:
Ok. Do you want to remove the "RETIRED" from the actual key, or the pub const declaring it?

Reply:
From both, because otherwise any retirement of an attribute becomes a breaking change. We can put the "RETIRED" notice as a doc comment on the const variables.